### PR TITLE
Add HW23

### PR DESCRIPTION
**Pull request by Gritskov-Vitaliy**

### CODE REVIEW

File: HW23/roles/hw23/tasks/hw23_1.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  ansible.builtin.service:
+    name: "{{  item  }}"
+    state: started
+    enabled: yes
+  become: true
+  loop:
+    - docker
+    - nginx
+
+- name: Var rc=0
```
Comment (mavrik90) [inline HW23/roles/hw23/tasks/hw23_1.yml]: For task 3b please use **register**  and **when**. For that you can create two bash command to generate code 0 and 2.

**Comment by mavrik90** [comment]
What about task 1 **Difference between ansible module: command, shell and raw**?
@Gritskov-Vitaliy

File: HW23/roles/hw23/tasks/hw23_1.yml (hunk 1 above)
Comment (Gritskov-Vitaliy) [reply HW23/roles/hw23/tasks/hw23_1.yml:38]: Command: The command(s) will not be processed through the shell, so variables like $HOSTNAME and operations like "*", "<", ">", "|", ";" and "&" will not work. Use the ansible.builtin.shell module if you need these features.
Shell: It is almost exactly like the ansible.builtin.command module but runs the command through a shell (/bin/sh) on the remote node.
Raw: Executes a low-down and dirty SSH command, not going through the module subsystem. This is useful and should only be done in a few cases. A common case is installing python on a system without python installed by default. Another is speaking to any devices such as routers that do not have any Python installed. In any other case, using the ansible.builtin.shell or ansible.builtin.command module is much more appropriate.

**Comment by Gritskov-Vitaliy** [comment]
> What about task 1 **Difference between ansible module: command, shell and raw**?
> @Gritskov-Vitaliy

Command: The command(s) will not be processed through the shell, so variables like $HOSTNAME and operations like "*", "<", ">", "|", ";" and "&" will not work. Use the ansible.builtin.shell module if you need these features.
Shell: It is almost exactly like the ansible.builtin.command module but runs the command through a shell (/bin/sh) on the remote node.
Raw: Executes a low-down and dirty SSH command, not going through the module subsystem. This is useful and should only be done in a few cases. A common case is installing python on a system without python installed by default. Another is speaking to any devices such as routers that do not have any Python installed. In any other case, using the ansible.builtin.shell or ansible.builtin.command module is much more appropriate.

File: HW23/roles/hw23/tasks/hw23_1.yml (hunk 1 above)
Comment (Gritskov-Vitaliy) [reply HW23/roles/hw23/tasks/hw23_1.yml]: Done
image

**Review by mavrik90 — APPROVED** [review]
LGTM!

**Pull request merged**